x-pack/plugins/apm/public/components/shared/charts/spark_plot/index.tsx

```diff
@@ -32,18 +37,60 @@ const flexGroupStyle = { overflow: 'hidden' };
 
 export function SparkPlot({
   color,
+  seriesLoading,
```

Comment:
What do you think of naming it `isLoading` instead?

Reply:
Good shout, I've changed it in https://github.com/elastic/kibana/pull/128196/commits/4504d9e31ed68a4ce50d0b46656d46855104b176